modules/aws_nodegroup/tf_module/main.tf

```diff
@@ -66,6 +66,15 @@ resource "aws_eks_node_group" "node_group" {
     min_size     = var.min_nodes
   }
 
+  dynamic "taint" {
+    for_each = var.taints
+    content {
+      key    = taint.value["key"]
+      effect = lookup(taint.value, "effect", "NO_SCHEDULE")
+      value  = lookup(taint.value, "value", "opta")
```

Comment:
Can we exclude the value if its not set by the user?

Reply:
it looks like taints are always present in the official examples, and I wanted to keep things uniform